composer.json

```diff
@@ -12,10 +12,12 @@
             "email": "will.knauss@gmail.com"
         }
     ],
-    "autoload": {
-        "classmap": [
-            "."
-        ]
+    "autoload":{
+        "psr-4":{
+            "ParseCsv\\": "src",
+            "ParseCsv\\extensions\\": "src\\extensions",
+            "ParseCsv\\tests\\": "tests"
```

Comment:
should be moved to section "autoload-dev"

Reply:
Yeah, you're right. I saw this now in a lot of open source libraries. Fixed in https://github.com/itexia/parsecsv-for-php/pull/6.